inflate.c

```diff
@@ -1140,7 +1140,10 @@ int32_t Z_EXPORT PREFIX(inflateSetDictionary)(PREFIX3(stream) *strm, const uint8
 
     /* copy dictionary to window using updatewindow(), which will amend the
        existing dictionary if appropriate */
+    int wrap_val = state->wrap; // Temporarily save to disable checksum in this function
+    state->wrap = 0;
```

Comment:
This contradicts the commit message... Essentially you should AND `state->wrap` with bit-inverse of `4` to just clear the relevant bit.

Reply:
We could but we know the control flow here doesn't check state->wrap for anything other than this and it's simpler, less error prone, and faster to wholesale copy the whole value and write it back.

Reply:
Maybe then update the commit message to rephrase the conflicting part. Saves from adding yet another comment here explaining why clearing whole variable is better than clearing just one bit.

Reply:
Sure, will fix.